src/dto/holdout-dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IShardRange } from './variation-dto';
+
+interface IHoldoutKey {
+  statusQuoShardRange: IShardRange;
+  shippedShardRange: IShardRange | null;
```

Comment:
null if no shipped variation (i.e. the experiment is still running)

Reply:
Do we still want to log this holdout user was in the not-status-quo part of the holdout test should they be bucketed into the holdout test?